app/src/main/res/layout/fragment_sidebar_main.xml

```diff
@@ -236,6 +236,13 @@
                         android:text="@string/fragment_sidebar_main_logout_title"
                         android:textAppearance="?attr/textAppearanceListItemSmall" />
             </LinearLayout>
+
+            <TextView
+                    android:id="@+id/version_info"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
```

Comment:
Since it is not a permanent solution, I'll just add the `?attr/listPreferredItemHeight` here.